spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -176,128 +176,38 @@ def mock_column(name, type, options = {})
   end
 
   describe '.get_schema_info' do
-    subject do
-      AnnotateModels.get_schema_info(klass, header)
-    end
-
-    let :klass do
-      mock_class(:users, primary_key, columns, indexes, foreign_keys)
-    end
-
-    let :indexes do
-      []
-    end
-
-    let :foreign_keys do
-      []
-    end
-
-    context 'when header is "Schema Info"' do
-      let :header do
-        'Schema Info'
+    context 'when option is not present' do
+      subject do
```

Comment:
Any reason why is `subject` under `context` instead of `describe`?

Reply:
@drwl Thanks, I'd like to split large commits into smaller ones.

I use `context` and `describe` depending on following link.
https://lmws.net/describe-vs-context-in-rspec

Are you getting at that `subject` under `context` is strange and `subject` should be under `describe`?

Reply:
Yep, I've usually seen `subject` be under `describe` instead of `context` but there doesn't seem to be anything on the style guide against it.

https://github.com/rubocop-hq/rspec-style-guide#subject-naming-in-context

Has `subject` under multiple contexts so seems to be fine.

Reply:
@drwl All right. I'll use your reply as a reference in the next PR. Thank you.